### DISCUSSION TITLE
Searching tool for (possibly old) HTML archives

### DISCUSSION THREAD

**Discussion by BillyLapworth** [Tyrrrz/DiscordChatExporter#659] [General]
[https://billswax.com/sites/discordexplorer/](https://billswax.com/sites/discordexplorer/)
I created a searching tool in vanilla JS for some of my old HTML Discord archives. They were very large, so using CTRL+F with a browser wasn't feasible, hence this. Hopefully you might get some use out of it, I could even create a public repo of it or something.

**Comment by Tyrrrz** [comment]
Nice :) I'm sure it might be useful for some
